spec/models/student_loans_spec.rb

```diff
@@ -22,4 +22,14 @@
       expect(StudentLoans.first_eligible_qts_award_year(AcademicYear.new(2030))).to eq AcademicYear.new(2019)
     end
   end
+
+  describe ".current_financial_year" do
+    it "returns a human-friendly string for the financial year the policy is currently accepting claims for" do
+      expect(policy_configuration.current_academic_year).to eq AcademicYear.new("2025/2026")
```

Comment:
I'm a bit confused as to _why_ this returns that particular Academic Year, as there doesn't seem to be any specific teardown/setup in this test. Can we make this a bit clearer?

Reply:
It's the currently-configured academic year [as defined the fixtures for policy configurations](https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/spec/fixtures/policy_configurations.yml#L3).

So it's less an expectation on behaviour and more an assertion on the current state of the system/data and also tells the reader what the currently configured year is so they understand how the expected value for financial year relates to it. I can see though that it's not particularly clear where it's coming from though. I've replaced this and the other expectation with a comment instead.

Reply:
Cool 👍